src/fixup.js

```diff
@@ -194,7 +198,18 @@
           button.innerText = (isOpen) ? '\u25BE collapse' : '\u25B4 expand';
         }
       }
+
       document.body.appendChild(node);
+      button.focus();
+      window.onkeydown = function (event) {
+        if (event.keyCode == 27 && !node.classList.contains("outdated-collapsed")) button.click();
+      }
+      document.addEventListener("focus", function(event) {
+        if (!node.classList.contains("outdated-collapsed") && !node.contains(event.target)) {
```

Comment:
nit: easiest to read and maintain:

```JS
var containsTarget = node.contains(event.target);
var isCollapsed = node.classList.contains("outdated-collapsed");
if (!isCollapsed && !containsTarget) {
```

Also, might be misunderstanding, but don't you want to do this if it contains the target?

Reply:
The "Keyboard interaction" section from https://www.w3.org/TR/wai-aria-practices-1.1/#keyboard-interaction-6 says the focus should stay inside the dialog.
If the element isn't contained in the `node`, the code just focuses on the `node` itself.

Reply:
Thanks for clarifying.